docs/source/algorithms/AddLogSmoothed-v1.rst

```diff
@@ -0,0 +1,54 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm performs a smoothing operation over the values in the sample log.
+
+The options available are boxcar (moving average) smoothing, FFT smoothing by zeroing, or FFT smoothing with a Butterworth filter.
+
+Usage
+-----
+
+**Example: Smoothing a log with moving average smoothing**
```

Comment:
Maybe not necessary, but this is a nice example of when a [plot directive](https://matplotlib.org/stable/api/sphinxext_plot_directive_api.html) could illustrate what this algorithm does really well.